lib/unhangout-routes.js

```diff
@@ -491,18 +549,51 @@ module.exports = {
             showEventOverview(req, res, "admin.ejs");
         });
 
-        app.get("/myevents/", ensureAuthenticated, function(req, res) {
+        app.get("/myevents/", ensureAuthenticated, utils.ensureAdmin,function(req, res) {
             showEventOverview(req, res, "myevents.ejs");
         });
 
+        app.post("/myevents/", ensureAuthenticated, utils.ensureAdmin, function(req, res) {
+
+            var eventID = req.body.eventId; 
+
+            var event = db.events.get(req.body.eventId);
+
+            if (!event) {
+                return res.send(400, "Missing event.");
+            }
+
+            if(req.body.admins) {
+                for (var c = 0; c< req.body.admins.length; c++) {
+                    
+                    var user = db.users.get(req.body.admins[c]);
+
+                    if (!user) { 
+                        return res.send(400, "Unrecognized user"); 
+                    }
+
+                    if(req.body.action == 'add-event-admin') {
+                        event.addAdmin(user);
+                        event.save();
```

Comment:
A minor thing, but this will save n times for n admins, and we only need to save once.  Might be preferable to move the save action out of the loop.